pkg/app/master/commands/xray/handler.go

```diff
@@ -1268,18 +1268,30 @@ func printObject(xc *app.ExecutionContext, object *dockerimage.ObjectMetadata) {
 	if object.Hash != "" {
 		hashInfo = fmt.Sprintf(" hash=%s", object.Hash)
 	}
-
-	fmt.Printf("%s: mode=%s size.human='%v' size.bytes=%d uid=%d gid=%d mtime='%s' %s%s '%s'",
-		object.Change,
-		object.Mode,
-		humanize.Bytes(uint64(object.Size)),
-		object.Size,
-		object.UID,
-		object.GID,
-		object.ModTime.UTC().Format(time.RFC3339),
-		objectHistoryString(object.History),
-		hashInfo,
-		object.Name)
+	xc.Out.Info("params",
```

Comment:
will be better to use "object" instead of "params" here (better aligns with the data we are displaying)

Reply:
Sounds right, thanks!